documentation/release_notes.markdown

```diff
@@ -10,329 +10,158 @@ canonical: "/puppetserver/latest/release_notes.html"
 
--   [SERVER-1644](https://tickets.puppetlabs.com/browse/SERVER-1644)
+As of Puppet 5.0, Puppet no longer writes node YAML files to its cache by default. This cache has been used in workflows where external tooling needs a list of nodes, but PuppetDB is now the preferred source of node information.
 
-## Puppet Server 2.7.0
+To retain the Puppet 4.x behavior, add the [`puppet.conf`](./configuration.markdown) setting `node_cache_terminus = write_only_yaml`. Note that `write_only_yaml` is deprecated, and users are encouraged to migrate to PuppetDB in order to retrieve node information.
 
-Released November 8, 2016.
+-   [SERVER-1819](https://tickets.puppetlabs.com/browse/SERVER-1819)
+-   [PUP-6060](https://tickets.puppetlabs.com/browse/PUP-6060)
 
-This is a feature and bug-fix release of Puppet Server.
+### New Feature: Performance metrics
 
-> **Warning:** If you're upgrading from Puppet Server 2.4 or earlier and have modified `bootstrap.cfg`, `/etc/sysconfig/puppetserver`, or `/etc/default/puppetserver`, see the [Puppet Server 2.5 release notes first](#potential-breaking-issues-when-upgrading-with-a-modified-bootstrapcfg) **before upgrading** for instructions on avoiding potential failures.
+Puppet Server 5.0 includes metrics support previously released in Puppet Enterprise, including [Grafana and Graphite support](./puppet_server_metrics.markdown), both [PE-style](./metrics-api/v1/metrics_api.markdown) and [Jokola-powered](./metrics-api/v2/metrics_api.markdown) metrics API endpoints, and the developer dashboard.
 
-### New Feature: Disable update checking and telemetry data collection
+-   [SERVER-1797](https://tickets.puppetlabs.com/browse/SERVER-1797)
+-   [SERVER-1752](https://tickets.puppetlabs.com/browse/SERVER-1752)
+-   [SERVER-1739](https://tickets.puppetlabs.com/browse/SERVER-1739)
+-   [SERVER-1737](https://tickets.puppetlabs.com/browse/SERVER-1737)
+-   [SERVER-1721](https://tickets.puppetlabs.com/browse/SERVER-1721)
+- [SERVER-1266](https://tickets.puppetlabs.com/browse/SERVER-1266)
+- [SERVER-1262](https://tickets.puppetlabs.com/browse/SERVER-1262)
+- [SERVER-1261](https://tickets.puppetlabs.com/browse/SERVER-1261)
+- [SERVER-1260](https://tickets.puppetlabs.com/browse/SERVER-1260)
+- [SERVER-1259](https://tickets.puppetlabs.com/browse/SERVER-1259)
 
-Puppet Server automatically communicates with Puppet's servers to check for updates. Puppet Server 2.7 adds the option to stop checking for updates by creating a new configuration file, [`product.conf`][product.conf], setting `check-for-updates` to false, then [restarting Puppet Server](./restarting.markdown).
+### New Feature: Optional support for JRuby 9k
 
-For more information, see the [`product.conf`][product.conf] documentation.
+Puppet Server 5.0 packages include the dependencies for both JRuby 1.7 (running Ruby language version 1.9.3) and JRuby 9k (running Ruby language version 2.3 or later). Puppet Server 5.0 uses JRuby 1.7 by default.
```

Comment:
Do you think it would be worth mentioning somewhere in the release notes that the packages are about 30 MB larger now because of the JRuby 9k dependencies that we're bundling in?  We mention this in the detailed notes for enabling JRuby 9k but it may be good to mention this here as well since this is something that users might notice / be curious about right away as they are upgrading.